packages/Discretization/test/tstMeshGenerator.cpp

```diff
@@ -10,158 +10,146 @@
 #include "MeshGenerator.hpp"
 #include <Teuchos_UnitTestHarness.hpp>
 
+// The out and success parameters come from the Teuchos unit testing macro
+// expansion.
+std::tuple<std::vector<std::vector<Coordinate>>, std::vector<unsigned int>>
+read_input_file( std::string const &filename, bool &success,
+                 Teuchos::FancyOStream &out )
```

Comment:
camelCase :p

Reply:
Also do you reckon it fits better in the test or in `MeshGenerator.hpp` header file?

Reply:
This is not enough to generate a useful mesh because the topology is missing. If you want, I can add this capability in the function and move it to `MeshGenerator.hpp`.

Reply:
No that's OK